src/olympia/versions/models.py

```diff
@@ -645,6 +645,21 @@ def was_auto_approved(self):
         return False
 
 
+class VersionPreview(BasePreview, ModelBase):
```

Comment:
I don't see the migration for this. (when you add it, make sure to create the indexes as well, including the one for the default ordering you have below)

Reply:
🙄 I always forget the migrations

Reply:
https://github.com/mozilla/addons-server/pull/7766/commits/cb70f154df3ef44edc74ee9cafe98ae098fbdc9c